packages/appstore/src/constants/tour-steps-config-new.tsx

```diff
@@ -0,0 +1,144 @@
+import { Step, Styles, Locale } from 'react-joyride';
+import React from 'react';
+import { Text, Button, Icon } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import 'Components/toggle-account-type/toggle-account-type.scss';
+
+export const tour_step_config: Step[] = [
+    {
+        title: (
+            <Text as='p' weight='bold' color='brand-red-coral'>
+                {localize('Switch accounts')}
+                <div className='toggle-account-type__divider' />
+            </Text>
+        ),
+        content: <Text as='p'>{localize('You can switch between real and demo accounts.')}</Text>,
+        target: '.account-type-dropdown--parent',
+        disableBeacon: true,
+    },
+    {
+        title: (
+            <Text as='p' weight='bold' color='brand-red-coral'>
+                {localize('Non-EU and EU regulators')}
+                <div className='toggle-account-type__divider' />
+            </Text>
+        ),
+        content: (
+            <Text as='p'>
+                <Localize
+                    i18n_default_text='You can create real accounts with non-EU and EU regulators. Click the <0><0/> icon to learn more about these accounts.'
+                    components={[
+                        <Text key={0}>
+                            <Icon icon='IcInfoOutline' />
+                        </Text>,
+                    ]}
+                />
+            </Text>
+        ),
+
+        target: '.regulators-switcher',
+        disableBeacon: true,
+    },
+    {
+        title: (
+            <Text as='p' weight='bold' color='brand-red-coral'>
+                {localize(`Trader's hub tour`)}
+                <div className='toggle-account-type__divider' />
+            </Text>
+        ),
+        content: <Text as='p'>{localize(`Click here if you ever need to repeat this tour.`)}</Text>,
+
+        target: '.trading-hub-header__tradinghub--onboarding--logo',
+        disableBeacon: true,
+    },
+];
+
+export const eu_tour_step_config: Step[] = [
+    {
+        title: (
+            <Text as='p' weight='bold' color='brand-red-coral'>
+                {localize('Switch accounts')}
+                <div className='toggle-account-type__divider' />
+            </Text>
+        ),
+        content: <Text as='p'>{localize('Switch between your demo and real accounts.')}</Text>,
+        target: '.toggle-account-type__button',
+        disableBeacon: true,
+        placement: 'bottom-end',
+    },
+    {
+        title: (
+            <Text as='p' weight='bold' color='brand-red-coral'>
+                {localize('Trading hub tour')}
+                <div className='toggle-account-type__divider' />
+            </Text>
+        ),
+        content: (
+            <Text as='p'>{localize(`Need help moving around?\n\nWe have a short turorial that might help.`)}</Text>
```

Comment:
new lines will fail the translation in Crowdin, try to define a component.

Reply:
Done.